src/sass/Fabric.Icons.Output.scss

```diff
@@ -34,6 +34,10 @@
   @include ms-Icon--l;
 }
 
+// @todo: 'DynamicsCRMLogo' class is deprecated, will be removed in a future release
+// @todo: 'CarotRightSolid8' class is deprecated, will be removed in a future release
```

Comment:
Did you want to move both the comment and the deprecated selector to the top? I think you wanted to be able to copy the newly generated classes over top without losing anything, right?